x-pack/plugins/apm/server/routes/services/get_services/get_service_transaction_stats.ts

```diff
@@ -70,13 +71,17 @@ export async function getServiceTransactionStats({
   randomSampler,
   documentType,
   rollupInterval,
+  useDurationSummary,
 }: AggregationParams): Promise<ServiceTransactionStatsResponse> {
   const outcomes = getOutcomeAggregation(documentType);
 
   const metrics = {
     avg_duration: {
       avg: {
-        field: getDurationFieldForTransactions(documentType),
+        field: getDurationFieldForTransactions(
+          documentType,
+          useDurationSummary
+        ),
```

Comment:
Maybe not for now, but I think we should split up `getDurationFieldForTransactions`. Either with function overloads of separate functions:

```ts
function getDurationFieldForTransactionType(
  type:
    | ApmDocumentType.ServiceTransactionMetric
    | ApmDocumentType.TransactionMetric
    | ApmDocumentType.TransactionEvent,
  useDurationSummaryField: boolean
)

function getDurationFieldForTransactions(searchAgggregatedTransactions: boolean) 
```